solidity/random-beacon/contracts/test/SortitionPoolStub.sol

```diff
@@ -8,6 +8,7 @@ import "../RandomBeacon.sol";
 // Stub contract used in tests
 contract SortitionPoolStub is ISortitionPool {
```

Comment:
Yep I'm aware but this should be done at once to avoid build failures 🤷‍♂️ I posted a TODO (in https://github.com/keep-network/keep-core/pull/2707) informing this stub is deprecated and real SP should be used.

Reply:
Yeah, it's not something we can do in this PR. I'll just keep complaining in every single PR until it's done so that we can all feel the pain lol.